architecture/service_catalog/service_catalog_cli.adoc

```diff
@@ -0,0 +1,439 @@
+*Google Compute Engine*
+For Google Cloud Platform, run the following command to setup firewall rules to allow incoming traffic:
+
+[source, bash]
+----
+$ gcloud compute firewall-rules create allow-service-catalog-secure --allow tcp:30443 --description "Allow incoming traffic on 30443 port."
+----
+
+[[service-catalog-cli-usage]]
+== Using `svcat`
+This section includes common commands to handle the user associated tasks listed
+in xref:index.adoc#service-catalog-design[the service catalog workflow]. Use the
+`svcat --help` command to get more information and view other available
+command-line options. The sample output in this section assumes that the Ansible
+Service Broker is already installed on the cluster.
+
+[[service-catalog-cli-broker-details]]
+=== Get broker details
+You can view a list available brokers, sync the broker catalog, and get details
+about brokers deployed in the service catalog.
+
+[[service-catalog-cli-find-broker]]
+==== Find brokers
+To view all the brokers installed on the cluster, run the following command:
+[source, bash]
+----
+$ svcat get brokers
+           NAME                                                        URL                                              STATUS
++-------------------------+-------------------------------------------------------------------------------------------+--------+
+  ansible-service-broker    https://asb.openshift-ansible-service-broker.svc:1338/ansible-service-broker                Ready
+  template-service-broker   https://apiserver.openshift-template-service-broker.svc:443/brokers/template.openshift.io   Ready                                       Ready
+----
+
+[[service-catalog-cli-sync-broker]]
+==== Sync broker catalog
+To refresh the catalog metadata from the broker:
+[source, bash]
+----
+$ svcat sync broker ansible-service-broker
+Synchronization requested for broker: ansible-service-broker
+----
+
+[[service-catalog-cli-view-broker]]
+==== View broker details
+To view the details of the broker, run the following command:
+[source, bash]
+----
+$  svcat describe broker ansible-service-broker
+  Name:     ansible-service-broker
+  URL:      https://openshift-automation-service-broker.openshift-automation-service-broker.svc:1338/openshift-automation-service-broker/
+  Status:   Ready - Successfully fetched catalog entries from broker @ 2018-06-07 00:32:59 +0000 UTC
+----
+
+[[service-catalog-cli-view-service-classes-and-plans]]
+=== View service classes and service plans
+When you create a `ClusterServiceBroker` resource, the service catalog
+controller queries the broker server to find all services it offers and creates
+a service class (`ClusterServiceClass`) for each of those services.
+Additionally, it also creates service plans (`ClusterServicePlan`) for each of
+the broker’s services.
+
+[[service-catalog-cli-view-serviceclass]]
+==== View service classes
+To view the available ClusterServiceClass resources:
+[source, bash]
+----
+$ svcat get classes
```

Comment:
Sorry. This part need to update. The new version svcat doesn't include UUID in get class|plan output.
```
$ svcat get classes
        NAME                   DESCRIPTION            
+-------------------+--------------------------------+
  rh-mediawiki-apb    Mediawiki apb implementation    
  
  ...
  
  rh-mariadb-apb      Mariadb apb implementation      
  rh-mysql-apb        Software Collections MySQL APB  
  rh-postgresql-apb   SCL PostgreSQL apb              
                      implementation  
```